src/class-main.php

```diff
@@ -37,6 +37,20 @@
  */
 class Main extends Lib_Core {
 
+  /**
+   * Returns the project environment variable descriptor.
+   *
+   * Used to cache DI container depending on the environment of the project.
```

Comment:
This is just one usage for it atm, but it can be used for other things too, so I'd add this as one of the uses.

Reply:
Sure but in this class-main.php it is used for DI container only but yes you can use it for some other configurations in the main class